iam: add tags support for users to the provider (#98)
C2Devel/terraform-provider-croccloud: website/docs/r/iam_user.md

* `password` - (Optional, Editable) The password of the user.
  If no value is specified, the password will be generated automatically.
* `phone` - (Optional, Editable) The phone number of the user.

> * `tags` - (Optional, Editable) Key-value map of tags for the IAM user.

Reviewer: ```suggestion
* `tags` - (Optional, Editable) Map of tags to assign to the user.
```

Там еще есть стандартная приписка про блок `default_tags`. Посмотри пожалуйста как в других ресурсах и добавь

> ## Attribute Reference

Reviewer: В атрибутах не хватает записи для `tags_all`

> * `tags` - (Optional, Editable) Map of tags to assign to the user. If configured with a provider [`default_tags` configuration block][default-tags] present, tags with matching keys will overwrite those defined at the provider-level.

Reviewer: Для использования метки [default-tags] ее надо объявить в начале страницы с соответствующей ссылкой. Посмотри как сделано на других страничках

> * `tags` - (Optional, Editable) Map of tags to assign to the user. If configured with a provider [`default_tags` configuration block][default-tags] present, tags with matching keys will overwrite those defined at the provider-level.

Reviewer: ```suggestion
* `tags` - (Optional, Editable) Map of tags to assign to the user. If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.
```

и формулировка будет такая) (это я тебя синхронизирую с соседним пром по переформатированию доки)

* `last_login_date` - The time in [RFC3339 format] when the user last logged in to the web interface.
* `login` - The login of the user.
* `secret_key` - The secret key of the user.

> * `tags_all` - Map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block][default-tags].

Reviewer: ```suggestion
* `tags_all` - Map of tags assigned to the user, including those inherited from the provider [`default_tags` configuration block][default-tags].
```
